prow/gerrit/client/syncer.go

```diff
@@ -146,25 +150,78 @@ func (st *SyncTime) Update(newState LastSyncState) error {
 		}
 	}
 
-	if !changed {
-		return nil
+	st.val = targetState
+
+	// Pediodically sync time back to storage instead of writing after every
+	// single project is processed, as the I/O would become performance
+	// bottleneck when there are lots of projects.
```

Comment:
I think this should instead belong in the commit message if it isn't there already. Describe the old situation, why that was bad, and explain the solution. This is because the comment "instead of writing after every single project is processed" is lost to history when we delete the old code in this PR. Instead this comment should only explain what is happening with the current code added in this PR.

Reply:
extended the commit message